src/components/Error.jsx

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+// eslint-disable-next-line react/prop-types
+const Error = ({ message }) => (
+  <div className=" p-2 text-white text-5xl">
+    Error:
+    {message?.message}
+  </div>
```

Comment:
- [ ] Kindly remove the `eslint-disable` statements used in this file. `Eslint` helps developers write clean, readable and industry-standard codes and using `eslint-disable` statements defeats this purpose. Kindly remove them. To solve the prop-types issue, you can read this [documentation](https://legacy.reactjs.org/docs/typechecking-with-proptypes.html). 👍 😎